src/bootstrap/test.rs

```diff
@@ -1418,14 +1418,10 @@ note: if you're sure you want to do this, please open an issue as to why. In the
             })
         };
         let lldb_exe = "lldb";
-        let lldb_version = Command::new(lldb_exe)
-            .arg("--version")
-            .output()
-            .map(|output| String::from_utf8_lossy(&output.stdout).to_string())
-            .ok();
+        let lldb_version = run(Command::new(lldb_exe).arg("--version"));
```

Comment:
This ```--version``` change isn't strictly necessary, but I figured I'd make ```lldb --version``` consistent with ```lldb -P```. It seems like ```run``` is appropriate to use in both places?

My lldb is actually failing here too, so really it shouldn't even be trying to do ```lldb -P```, which it no longer would with this change.

I don't have a working lldb to test this.